lib/pos_blockchain.go

```diff
@@ -364,7 +364,7 @@ func (bc *Blockchain) getLineageFromCommittedTip(desoBlock *MsgDeSoBlock) ([]*Bl
 		if currentBlock.Hash.IsEqual(highestCommittedBlock.Hash) {
 			break
 		}
-		if currentBlock.CommittedStatus == COMMITTED {
+		if IsBlockCommitted(currentBlock) {
```

Comment:
nit: maybe differentiate naming between blocks (of type MsgDeSoBlock) from block nodes (of type BlockNode). In this case, we can make it a bit more obvious that the committed check is just checking the status on the node rather than checking if the block itself is committed in the chain through a lookup underneath
```suggestion
		if currentBlockNode.IsCommitted() {
```

Reply:
Will address naming conventions in a separate PR